libs/utils/src/getEasSupportedNetwork.ts

```diff
@@ -0,0 +1,52 @@
+import type { BlockchainNetwork } from "./types"
+
+export const easCredentialSupportedNetworks: BlockchainNetwork[] = [
```

Comment:
What about creating this exact constant inside the `getSupportedNetworks.ts` file instead of a new file?

Reply:
Noted, will update now.